src/Parser/Parser.hs

```diff
@@ -176,18 +188,13 @@ notAlreadyInUse x = do
     then unexpected $ "redefinition of " ++ parsed
     else return parsed
 
--- | Valid character set that can be used to describe a game (after the first lead character)
-gameIdentifierChars :: [Char]
-gameIdentifierChars = ['a'..'z']++['A'..'Z']++['0'..'9']++"_"
```

Comment:
note that `identifier` is a parser that corresponds exactly to this list since we set it to be like Haskell's identifiers

Reply:
That comment was made carelessly and is _quite_ false.  I'm back to correct the record! `identifier` will only accept something starting with a letter and of course, it excludes all the reserved names. To say it corresponds _exactly_ to this list is, err, inexact.